src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+
+  --link-color: #00acdc;
+  --header-heihgt: 60px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-self: center;
+  padding: 0 50px;
+}
+
+.logo {
+  padding: 10px 0;
+}
+
+.nav_list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  align-self: center;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  text-transform: uppercase;
+  position: relative;
```

Comment:
we can move it to our modifier because we only need positioning for the element that has the modifier and only on it.